webapp/docs/add-new-token-to-stake.md

```diff
@@ -0,0 +1,127 @@
+# Adding a New Token to Staking
+
+The goal of this guide is to walk you through the steps needed to add a new token to the staking list.
+
+## Before you begin
+
+You need to add the new token to the [hemilabs/token-list](https://github.com/hemilabs/token-list) repository before adding the token to stake here. You can follow this [tutorial](https://github.com/hemilabs/token-list/blob/master/docs/add-new-token.md) to do that.
+
+Proceed with this tutorial only when the token is available in a released version of [hemilabs/token-list](https://github.com/hemilabs/token-list/releases) npm package.
+
+## Step 1 - Bump the Token List package version
+
+Update the [hemilabs/token-list](https://github.com/hemilabs/token-list) package version in `webapp/package.json` file for the version that have the token you want to add to stake.
+
+```jsonc
+"dependencies": {
+    ...
+-   "@hemilabs/token-list": "1.15.0",
++   "@hemilabs/token-list": "1.16.0",
+    ...
+}
+```
+
+## Step 2 - Add the token to stakeTokens file
+
+Add the new token to the [stakeTokens.ts](../tokenList/stakeTokens.ts) file in the `stakeWhiteList` object and under the appropiate chain id (mainnet or testnet) as below:
+
+```ts
+// webapp/tokenList/stakeTokens.ts
+
+export const stakeWhiteList: Partial<
+  Record<Token['chainId'], Record<string, StakeExtensions>>
+> = {
+    [hemi.id]: {
+        ...
+        // satUSD
+        '0xb4818BB69478730EF4e33Cc068dD94278e2766cB': {
+            protocol: 'satoshi',
+            priceSymbol: 'usdc',
+            rewards: ['hemi3x', 'satoshi', 'pell'],
+            website: 'https://www.satoshiprotocol.org',
+        },
+        ...
+    },
+    ...
+}
+```
+
+## Step 3 - Add the protocol logo image (if needed)
+
+If the new token protocol is not already in the [protocolImages.ts](../app/[locale]/stake/protocols/protocolImages.ts) file you need to add the protocol logo image to the [/stake/protocols/images](webapp/app/[locale]/stake/protocols/images) folder and add it to the `protocolImages` list as below:
```

Comment:
I think this link is broken

```suggestion
If the new token protocol is not already in the [protocolImages.ts](../app/[locale]/stake/protocols/protocolImages.ts) file you need to add the protocol logo image to the [/stake/protocols/images](../app/[locale]/stake/protocols/images) folder and add it to the `protocolImages` list as below:
```

(I haven't tested if that´s actually the fix, I think it is)

Reply:
fixed